src/components/Account/Account.scss

```diff
@@ -0,0 +1,96 @@
+@use "../../styles/variables" as *;
+@import url("https://fonts.googleapis.com/css2?family=Edu+NSW+ACT+Foundation:wght@400..700&family=Oswald:wght@200..700&display=swap");
```

Comment:
You should be able to remove this and the commented-out style as well `// justify-content: end;`